docker/run.sh

```diff
@@ -65,6 +70,7 @@ fi
 # Execute docker container run to start the Big Data Crawler Server
 docker container run --name big-data-crawler-server \
            --rm \
+           -p 8001:8001 \
```

Comment:
Since we've got nice long command line arguments (`--name`, `--rm`, `--volume`) may as well use `--publish` instead of `-p` here too 😇

Reply:
Thanks! I updated 2 related pull requests, using `--publish` instead of `-p`.